nvflare/private/fed/client/client_engine.py

```diff
@@ -53,7 +53,6 @@ def __init__(self, client, client_name, args, rank, workers=5):
         Args:
```

Comment:
why `ClientEngine` don't need `validate_targets`?

Reply:
There's no use case for the client parent process to validate_targets().